### ISSUE TITLE
Edit profile pic doesn't work.  You can see the mobile device working, but the app gets stuck at this point.

### ISSUE THREAD

**Issue by inder243** [brianhorakh/vita#8]
Unable to edit pofile picture,After updating picture, we are getting this in response:
{"success":true,"**sql":"update  Users set ProfilePic='2016-05-03/12/.profile-100.jpg' where Id='100'","now":"2016-05-03 05:25:57","ProfilePicFile":"2016-05-03/12/.profile-100.jpg","ProfilePicURL":"http://www.vitatheapp.com/files/2016-05-03/12/.profile-100.jpg"}

**Comment by brianhorakh** [comment]
Can you please send me the request you are using to update the picture?  Thanks!

**Comment by inder243** [comment]
![2016-05-31-112119_1366x768_scrot](https://cloud.githubusercontent.com/assets/7559712/15664522/3c727276-2722-11e6-83a4-52264a7eb14a.png)

**Comment by brianhorakh** [comment]
Not sure how else to ask this question.
The URL/File in the DB is not valid, so it's not going to work.
Can you please send me the API request you are using to upload the profile picture.
Thanks!

**Comment by inder243** [comment]
We are using this Web service to upload the Profile Pic : 
http://www.vitatheapp.com/vita-api/UserProfileUpdate?uid=100&authid=F1E78A8C-F63B-11E5-BC59-A7AA58C3F44C&file1=

*We are sending the image  encode To Base64String. The Image data is less than 1mb in all cases.

We are getting this response:

 Email = "Kavita.r@iapptechnologies.com";
    PasswordPlainText = qwerty;
    PhoneNumber = 56565656655;
    ProfilePic = "2016-05-31/13/.profile-100.jpg";
    ProfilePicURL = "http://www.vitatheapp.com/files/2016-05-31/13/.profile-100.jpg";
    Username = Kavi;
    now = "2016-06-01 22:06:50";
    success = 1;

Since the response success = 1,this mean image data is successfully delivered. But We are not able to open the ProfilePicURL.
It responding with  "404 Not Found".

**Comment by SeanGuerrero** [comment]
Brian, Please see the error message above.

**Comment by brianhorakh** [comment]
I can't imagine this would ever possibly work as an HTTP GET since the url would be too large.

I did find a bug when testing, and I was able to upload a file using the following 'curl' command line

 curl -v http://www.vitatheapp.com/vita-api/UserProfileUpdate --form uid=100 --form authid=F1E78A8C-F63B-11E5-BC59-A7AA58C3F44C -F ProfilePic=@tests/test.jpg

Please make sure you are using "ProfilePic" instead of "file1" parameter. Thanks!

http://www.vitatheapp.com/files/2016-06-09/23/image.profile-100.jpg

**Comment by SeanGuerrero** [comment]
Vakul, did you see this from Brian?

**Comment by inder243** [comment]
Yes, have seen that and we are already are using the same ProfilePic. But unable to get any image on URL.
![2016-06-20-114343_1366x768_scrot](https://cloud.githubusercontent.com/assets/7559712/16184754/825b68f0-36dc-11e6-8658-41531b48a47d.png)

**Comment by SeanGuerrero** [comment]
Vakul is this an issue that needs to be addressed by Brian?  

Brian can you please check this out?

**Comment by brianhorakh** [comment]
Sean - this is not my issue, I provided a curl example above.  They can't upload a file via "GET" they need to use a "POST" -- I think that's the issue.

**Comment by brianhorakh** [comment]
curl -v http://www.vitatheapp.com/vita-api/UserProfileUpdate --form uid=100 --form authid=F1E78A8C-F63B-11E5-BC59-A7AA58C3F44C -F ProfilePic=@tests/test.jpg
- About to connect() to www.vitatheapp.com port 80 (#0)
-   Trying 66.171.252.86... connected
- Connected to www.vitatheapp.com (66.171.252.86) port 80 (#0)
  > POST /vita-api/UserProfileUpdate HTTP/1.1
  > User-Agent: curl/7.19.7 (x86_64-redhat-linux-gnu) libcurl/7.19.7 NSS/3.19.1 Basic ECC zlib/1.2.3 libidn/1.18 libssh2/1.4.2
  > Host: www.vitatheapp.com
  > Accept: _/_
  > Content-Length: 13681
  > Expect: 100-continue
  > Content-Type: multipart/form-data; boundary=----------------------------7805d100f03d
  >
  < HTTP/1.1 100 Continue
  < HTTP/1.1 200 OK
  < Server: nginx/1.8.1
  < Date: Mon, 11 Jul 2016 20:24:49 GMT
  < Content-Type: text/html
  < Transfer-Encoding: chunked
  < Connection: keep-alive
  <
- Connection #0 to host www.vitatheapp.com left intact
- Closing connection #0
  {"success":true,"**sql":"update  Users set ProfilePic='2016-07-11/20/image.profile-100.jpg' where Id='100'","now":"2016-07-11 13:11:24","ProfilePicFile":"2016-07-11/20/image.profile-100.jpg","ProfilePicURL":"http://www.vitatheapp.com/files/2016-07-11/20/image.profile-100.jpg"}

**Comment by inder243** [comment]
done
